src/cli/cli.cpp

```diff
@@ -2722,6 +2722,18 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli log level
```

Comment:
Should be `@cli log level (get)`

Usually we'd combine get and set into one, but I think since you need a flag set to use the set version, we can keep them separate here.

Reply:
Actually, this can stay as `log level`, sorry.  We don't use `(get)` on its own with other commands.